[dependency] Vite 6 by default?

When install with `--force` flag, getting warning, when without flag, getting error sort. 

> And I hope you get a dependency bot someday. 

```
npm warn using --force Recommended protections disabled.
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: vite-plugin-static-copy@2.1.0
npm warn Found: vite@6.0.0
npm warn node_modules/vite
npm warn   peer vite@"^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0" from vite-plugin-solid@2.11.0
npm warn   node_modules/vite-plugin-solid
npm warn     dev vite-plugin-solid@"^2.11.0" from the root project
npm warn   2 more (vitefu, the root project)
npm warn
npm warn Could not resolve dependency:
npm warn peer vite@"^5.0.0" from vite-plugin-static-copy@2.1.0
npm warn node_modules/vite-plugin-static-copy
npm warn   vite-plugin-static-copy@"2.1.0" from the root project
npm warn
npm warn Conflicting peer dependency: vite@5.4.11
npm warn node_modules/vite
npm warn   peer vite@"^5.0.0" from vite-plugin-static-copy@2.1.0
npm warn   node_modules/vite-plugin-static-copy
npm warn     vite-plugin-static-copy@"2.1.0" from the root project
```

(Not related: also same issue with solidjs plugin)

--- a/.changeset/short-hairs-melt.md
+++ b/.changeset/short-hairs-melt.md
@@ -1,0 +1,5 @@
+---
+'vite-plugin-static-copy': minor
+---
+
+add Vite 6 to peer dep

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
   "homepage": "https://github.com/sapphi-red/vite-plugin-static-copy#readme",
   "sideEffects": false,
   "peerDependencies": {
-    "vite": "^5.0.0"
+    "vite": "^5.0.0 || ^6.0.0"
   },
   "devDependencies": {
     "@changesets/changelog-github": "^0.5.0",
@@ -60,7 +60,7 @@
     "tsup": "^8.3.5",
     "typescript": "^5.6.3",
     "typescript-eslint": "^8.13.0",
-    "vite": "^5.4.10",
+    "vite": "^6.0.1",
     "vitest": "^1.6.0",
     "vitest-github-actions-reporter": "^0.11.1"
   },

--- a/test/fixtures/package.json
+++ b/test/fixtures/package.json
@@ -13,7 +13,7 @@
     "build:structured": "vite --config vite.structured.config.ts build"
   },
   "devDependencies": {
-    "vite": "^5.4.10",
+    "vite": "^6.0.0",
     "vite-plugin-static-copy": "workspace:*"
   }
 }

